README.md

```diff
@@ -304,13 +315,19 @@ You will need to install the [Visual C++ Redistributable](https://support.micros
 #### With Chocolatey
 
 You can install `bat` via [Chocolatey](https://chocolatey.org/packages/Bat):
+
 ```bash
 choco install bat
 ```
 
+if you get an error like
+"Access to the path 'C:\ProgramData\chocolatey\lib\Less\legal' is denied."
+then run your comman prompt as administration (https://stackoverflow.com/questions/55600258/fix-chocolatey-access-is-denied)
```

Comment:
Typos:

"comman prompt"
"as administration" (should be "as administrator"?)

Reply:
sorry for typos, i will correct them